docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
 
 DTRs hold sensitive information: a SubmodelDescriptor may not give access to the actual Submodel-data but all in cumulo hint at 
 production volumes as each Twin represents an asset. Therefore, Catena-X implements decentral DTRs (DDTR), each running with a 
-business partner. [In an IDTA-Whitepaper](https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2023/06/Decentralized-Registries-Taxonomy-of-decentralized-registries-and-an-architectural-overview_.pdf ),
+business partner.
+[In an IDTA-Whitepaper](https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2023/06/Decentralized-Registries-Taxonomy-of-decentralized-registries-and-an-architectural-overview_.pdf ),
 several high-level concepts for DDTRs are introduced. The AAS-specification remains agnostic to the approaches and endorses
 none of them. Catena-X must deal with the additional complexity that stems from the interaction with the EDC.
 
 Leveraging the native capabilities of the EDC and the EDC Discovery Service, Catena-X uses a discovery pattern that has the
-same capability as a central [Digital Twin Registry](#digital-twin-registry): It allows to start a Discovery Process with only an AssetId and 
-As none of the options fit the data-space-specific requirements of Catena-X entirely, yet a different approach is implemented:
+same capability as a central [Digital Twin Registry](#digital-twin-registry): 
+It allows to start a Discovery Process with an Asset ID. 
 However, in Catena-X some of the data is deemed so sensitive that a central authority can't be
-trusted with it. Thus, each Data Provider will run their own DTR which poses a challenge for discovery. After all, a
+trusted with it. Thus, a decentralized approach is implemented: each Data Provider will run their own DTR. 
+This poses a challenge for discovery if the BPN of the supplier is not known by the data consumer. After all, a
 Data Consumer must still find out the address where to fetch the data from. That's why Catena-X has introduced a 
 three-step discovery pattern made up of the central microservices Discovery Finder, BPN Discovery (or several of them) 
-and finally the EDC discovery that is part of the CX-Portal. They are also part of this Kit.
+and finally the EDC discovery that is part of the CX-Portal. 
+The Discovery Finder and the BPN Discovery service are described as part of this Kit. 
```

Comment:
Again a space at the end of the line ;-)
If we already refer to the documentation of these services, shouldn't we directly provide a link?
```suggestion
The Discovery Finder and the BPN Discovery service are described as part of this Kit.
```